src/renderer/Core/ThemeContext.tsx

```diff
@@ -3,10 +3,12 @@ import { createContext } from "react";
 
 type ThemeContextProps = {
     theme: Theme;
+    isDarkMode: boolean;
```

Comment:
Can we call this `shouldPreferDarkColors` to stay consistent with other parts of the code?